grid2op/Opponent/RandomLineOpponent.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2019-2020, RTE (https://www.rte-france.com)
+# See AUTHORS.txt
+# This Source Code Form is subject to the terms of the Mozilla Public License, version 2.0.
+# If a copy of the Mozilla Public License, version 2.0 was not distributed with this file,
+# you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+# This file is part of Grid2Op, Grid2Op a testbed platform to model sequential decision making in power systems.
+
+import numpy as np
+from grid2op.Opponent import OpponentWithConverter
+from grid2op.Converter import LineDisconnection
+
+
+class RandomLineOpponent(OpponentWithConverter):
```

Comment:
Top priority: get rid of all unecessary complexity. Don't code on bases you are not familiar with. So remove all call to this `OpponentWithConverter`. Then start to code on fresh ground. It will avoid potential bugs due to a code you don't control.

Reply:
done